dango/warp/src/execute.rs

```diff
@@ -52,12 +61,26 @@ pub fn execute(ctx: MutableCtx, msg: ExecuteMsg) -> anyhow::Result<Response> {
     }
 }
 
+#[cfg_attr(not(feature = "library"), grug::export)]
+pub fn cron_execute(ctx: SudoCtx) -> anyhow::Result<Response> {
+    for (denom, rate_limit) in RATE_LIMIT
+        .range(ctx.storage, None, None, grug::Order::Ascending)
+        // Need to collect here because the iterator lock the ctx.storage.
+        .collect::<StdResult<Vec<_>>>()?
```

Comment:
In my issue I suggested putting all rate limits in a single `BTreeMap` so you don't need to do this.